app/models/rent.rb

```diff
@@ -0,0 +1,5 @@
+class Rent < ApplicationRecord
+  validates :user_id, :book_id, :initial_date, :final_date, presence: true
+  belongs_to :user
```

Comment:
In order to check for book presence you can use: `belongs_to :user, :book, required: true`